app/src/main/java/com/globant/practice/presentation/view/activity/HomeActivity.java

```diff
@@ -14,10 +25,22 @@
  */
 public class HomeActivity extends AppCompatActivity implements HomeView {
 
+    private RecyclerView listHomeRecyclerView;
+    private ProgressDialog fetchUserIndicator;
+    private SubscriberAdapter listHomeAdapter;
     @Inject
     HomePresenter presenter;
 
     /**
+     * Manages the actions when the user makes click
+     */
+    private SubscriberAdapter.OnUserClickListener userClickListener = new SubscriberAdapter.OnUserClickListener() {
```

Comment:
You can implement this interface in the activity so as to avoid creating a new attribute